src/tools/chdman.cpp

```diff
@@ -2810,6 +2810,13 @@ static void do_extract_cd(parameters_map &params)
 				}
 			}
 		}
+		else
+		{
+			cdrom_file::toc* trackinfo = (cdrom_file::toc*)&toc;
+						
+			for (int tracknum = 0; tracknum < toc.numtrks; tracknum++)
+				trackinfo->tracks[tracknum].splitframes = 0;
+		}
```

Comment:
It might be better to initialize this in `cdrom_file::parse_metadata` so that it'll properly be initialized in all cases instead of just this specific case.

Reply:
Scratch the part about the `parse_cue`/etc functions. I just checked and it looks like there's a `memset(&outtoc, 0, sizeof(outtoc));` for all of the parser functions, so loading the loose files should be ok. It looks like `parse_metadata` is the only one that isn't initializing the entire toc to 0 before using it.

Reply:
I scratched the part about the parse_cue and added `memset(&toc, 0, sizeof(toc));` to parse_metadata. Works also as expected. Hope my first pull request is correct now.